__TESTS__/unit/transformation/TransformableAsset.test.ts

```diff
@@ -0,0 +1,77 @@
+import TransformableImage from "../../../src/transformation/TransformableImage";
+import TransformableVideo from "../../../src/transformation/TransformableVideo";
+
+// List of assets types to test
+const assetTypes: any = {
+  TransformableImage,
+  TransformableVideo
+};
+
+// Dynamic asset generator
+class Asset {
+  constructor(className: string, opts: any=null) {
+    return new assetTypes[className](opts);
+  }
+}
+
+['TransformableImage', 'TransformableVideo'].forEach((assetClass) => {
```

Comment:
I'd like to avoid dynamically generating code like this, since it makes it difficult to find it later when something fails,

The failed test will be `Tests for TransformableImage` but if you try to search the project, you won't find such a test.

Reply:
There's a worst drawback for the alternative of duplicate tests: alignment